test/axes/timeAxisTests.ts

```diff
@@ -294,6 +294,34 @@ describe("TimeAxis", () => {
     svg.remove();
   });
 
+  it("tick labels show correctly when display format is set to 'center'", () => {
+    let svg = TestMethods.generateSVG(400, 100);
+    scale.domain([new Date("2015-09-02"), new Date("2015-09-03")]);
+    axis.tierLabelPositions(["center", "center"]);
+    axis.renderTo(svg);
+
+    let labels = axis.content().selectAll(`.${Plottable.Axis.TICK_LABEL_CLASS}`);
+    assert.notStrictEqual(labels.size(), 0, "More than one labels are selected in testing");
+
+    let axisBoundingRect: ClientRect = (<Element>axis.background().node()).getBoundingClientRect();
+    let isInsideAxisBoundingRect = function(innerRect: ClientRect) {
+        return (
+          Math.floor(axisBoundingRect.left) <= Math.ceil(innerRect.left) &&
+          Math.floor(axisBoundingRect.top) <= Math.ceil(innerRect.top) &&
+          Math.floor(innerRect.right) <= Math.ceil(axisBoundingRect.right) &&
+          Math.floor(innerRect.bottom) <= Math.ceil(axisBoundingRect.bottom)
+        );
+    };
+
+    labels.each(function(d, i) {
+      let labelVisibility = window.getComputedStyle(this).visibility;
+      let boundingClientRect = this.getBoundingClientRect();
+      let isInside = isInsideAxisBoundingRect(boundingClientRect);
+      assert.isFalse(isInside && (labelVisibility === "hidden"),  `Wrong labels ${i} are ${labelVisibility}`);
```

Comment:
Can we have a better assert message here?  If the test were to fail, the test would result to something like
"Wrong labels 3 are hidden."

This isn't really that helpful.

Let's take a step similar to the previous pull request and check the relationship between `isInside` and `labelVisibility === "hidden"` to get a better assert.

As a general rule of thumb, `assert.isTrue` / `assert.isFalse` is generally not at all useful unless testing for the simple cases.